src/scripts/ci_build.py

```diff
@@ -326,6 +331,10 @@ def sanitize_kv(some_string):
                 test_cmd += ['--pkcs11-lib=%s' % (pkcs11_lib)]
 
     if target in ['coverage', 'sanitizer']:
+        if target_os != 'windows' or target != 'sanitizer':
+            # GitHub Actions worker intermittently ran out of memory when
+            # asked to allocate multi-gigabyte buffers under MSVC's ASan.
+            test_cmd += ['--run-memory-intensive-tests']
```

Comment:
Let's invert this option (to `--skip-memory-intensive-tests`) because otherwise nobody (including eg an end user building on some weird CPU/compiler we've never seen) will have these tests run when they build locally, unless they know about this magic flag. That doesn't matter so much for this one particular RNG test but would be if it started being used to gate for instance an Argon2 test with very large memory.

Reply:
I considered that for the same reason but went with the 'disabled by default' to stay consistent with `--run-long-tests`. We could turn both around? Or just don't care about consistency here. I don't have a strong feeling either way.

Reply:
Eh. Not really strong feelings here I guess.

Let's make one minor change here: add `--run-memory-intensive-tests` to the default CI run and then *remove* it for the Windows sanitizer build only. That way we are still running the RNG 4GB test on macOS and Windows. Otherwise, AIUI, we are only ever running it within the Linux coverage build, so we could miss some problem that creeps into how we implement `System_RNG` on another platform.

Reply:
Done. For the record: The same is true for the "long" tests. `--run-long-tests` is only ever executed for the coverage build (and -- with this PR -- with sanitizers on all three platforms)

Reply:
Thanks for pointing that out. I'll address this.